Causality_analysis.py

```diff
@@ -0,0 +1,164 @@
+# -*- coding: utf-8 -*-
+
+subjects_dir = '/home/qdong/freesurfer/subjects/'
```

Comment:
@pravsripad Yes, you are right, I will make them in one file and make a function to set the pathes.